bin/gwdetchar-scattering

```diff
@@ -532,6 +541,67 @@ print(table)
 print('\n\n')
 table.write(summfile, overwrite=True)
 
+# -- launch omega scans -------------------------------------------------------
+
+nscans = min(args.omega_scans, len(table))
+if nscans > 0:
+    # launch scans
+    ind = random.sample(range(0, len(table)), nscans)
+    omegatimes = [str(t) for t in table['trigger_time'][ind]]
+    logger.debug('Collected {} event times to omega scan: {}'.format(
+        nscans, ', '.join(omegatimes)))
+    logger.info('Creating workflow for omega scans')
+    batch = [find_executable('gwdetchar-omega-batch')] + omegatimes + [
+        '--ignore-state-flags',
+        '--output-dir', 'scans',
+        '--ifo', args.ifo,
+        '--condor-timeout', str(4),
+        '--submit'
+    ]
+    logger.debug("$ {}".format(" ".join(batch)))
+    proc = subprocess.Popen(batch)
+    out, err = proc.communicate()
+    if proc.returncode:
+        raise subprocess.CalledProcessError(
+            proc.returncode, ' '.join(batch))
```

Comment:
can you just use `subprocess.check_call` here?

Reply:
@duncanmmacleod, aye, I've implemented this just now